src/docs/security-and-privacy-compliance/devops-security-considerations.md

```diff
@@ -322,133 +298,137 @@ Retention is set as follows:
+* bcgov-platform-services - git repository for platform services team
 
 These resources are available:
 - [BC Government organizations in GitHub](/bc-government-organizations-in-github/)
 
 **GitHub Apps**
 
-Teams may request GitHub apps to be associated with their own or all projects in a GitHub organization.  These requests are reviewed by a platform administrator for validity and scope.  These are also shared with Ministry security staff to ensure they are acceptable for connection.
+Teams may request GitHub apps to be associated with their own or all projects in a GitHub organization.  These requests are reviewed by a the Developer Experience team for validity and scope.  These are also shared with Ministry security staff to ensure they are acceptable for connection.
 
 **GitHub Enterprise**
 
 We currently use of GitHub Enterprise.  Contact the Developer Experience team for license information.
-- [GitHub Enterprise user licenses in the BC government](/github-enterprise-user-licenses-bc-government/)
-- https://github.com/enterprises/bcgov-ent/sso
+* [GitHub Enterprise user licenses in the BC government](/github-enterprise-user-licenses-bc-government/)
+- [Province of British Columbia Enterprise’s single sign-on provider](https://github.com/enterprises/bcgov-ent/sso)
 
-------
-### <a name="protected-c-use"></a>DevOps Team recommendations for Protected C data creation/storage/use
+## DevOps Team recommendations for Protected C data creation, storage and use
 
 This is a list of specific considerations for teams creating/storing/using Protected C data.  **This only applies to data on the Emerald Cluster.**
 
 Product teams should be aware that any creation/storage/use of Protected C data in any system should only be considered after discussion with Ministry Security Officer (MISO) and Ministry Privacy Officer (MPO).  They may require additional controls to those listed below.
  
 Protected C data should be encrypted at rest. 
-- The NetApp storage appliance used is NOT encrypted at its core, so this means that associate provisioned volume claims (PVCs) are not encrypted by default.  However, it is possible to request an encrypted volume claim. 
-  - This is done at pvc creation time using the following https://netapp-trident.readthedocs.io/en/stable-v21.07/kubernetes/operations/tasks/backends/ontap/ontap-san/configuration.html?highlight=encryption#configuration-options-for-provisioning-volumes.  
+- The NetApp storage appliance used is **not** encrypted at its core, so this means that associate provisioned volume claims (PVCs) are not encrypted by default.  However, it is possible to request an encrypted volume claim.
+
+  - This is done at [pvc creation time](https://netapp-trident.readthedocs.io/en/stable-v21.07/kubernetes/operations/tasks/backends/ontap/ontap-san/configuration.html?highlight=encryption#configuration-options-for-provisioning-volumes)
```

Comment:
I think this is a setting at the storage class level, not the pvc level. I don't think app teams currently can create an encrypted PVC.

Reply:
This was based on a convo we had and testing in the lab:  https://app.zenhub.com/workspaces/platform-experience-5bb7c5ab4b5806bc2beb9d15/issues/gh/bcdevops/developer-experience/1970.   If I got wording wrong I can change.

Reply:
Ya, we tested an encrypted storage class in the lab, but never rolled it out to prod. I think there wasnt much demand for it?

It would be a different storage class you'd request when creating a pvc, instead of `netapp-file-standard` youd pick `netapp-file-encrypted`.